kart/point_cloud/import_.py

```diff
@@ -142,15 +145,14 @@ def point_cloud_import(ctx, sources):
         repo.head_commit,
     )
 
-    # TODO: Don't accept all possible versions of everything / maybe convert everything to COPC-1.0
-    copc_version = copc_version_set[0]
-    if copc_version == 1:
-        kart_format = "pc:v1/copc-1.0"
-    elif copc_version != NOT_COPC:
-        kart_format = f"pc:DEV/copc-{copc_version}"
-    else:
-        filetype = "laz" if compressed_set[0] is True else "las"
-        kart_format = f"pc:DEV/{filetype}-{version_set[0]}"
+    is_copc = (
```

Comment:
should we silently support future variants of COPC? I'd expect them to work just as well as current versions, so I'd tend not to downgrade to LAZ 1.4/COPC1.0 here if we encounter a newer version.

Reply:
Ah I guess? There is no COPC > 1.0 (and no Kart Point Clouds really either) so hard for me to guess if those two non-existent things are compatible. But at least allowing it makes no real difference right now... I'll allow it